src/Essentials/src/FileSystem/FileSystem.ios.tvos.watchos.macos.cs

```diff
@@ -53,9 +53,11 @@ internal static string PlatformGetContentType(string extension)
 			// ios does not like the extensions
 			extension = extension?.TrimStart('.');
 
-			var id = UTType.CreatePreferredIdentifier(UTType.TagClassFilenameExtension, extension, null);
-			var mimeTypes = UTType.CopyAllTags(id, UTType.TagClassMIMEType);
-			return mimeTypes?.Length > 0 ? mimeTypes[0] : null;
+			// var id = UTType.CreatePreferredIdentifier(UTType.TagClassFilenameExtension, extension, null);
+			// var mimeTypes = UTType.CopyAllTags(id, UniformTypeIdentifiers.UTTagClass.MimeType.ToString());
+			// return mimeTypes?.Length > 0 ? mimeTypes[0] : null;
```

Comment:
Is there no replacement? This seems like something that we need?

Reply:
yeah i m still working on this, wasn't able to run the sample to test what i had